ansible/roles/debops.apt/templates/etc/ansible/facts.d/apt.fact.j2

```diff
@@ -15,7 +15,9 @@ import os
 {% set apt__tpl_default_sources = [] %}
 {% set apt__tpl_default_sources_map = {} %}
 {% set apt__tpl_source_distributions = {} %}
-{% for repo in apt__security_sources %}
+{% for repo in ( apt__host_security_sources,
+                 apt__group_security_sources,
+                 apt__security_sources ) %}
```

Comment:
With this change, `apt__tpl_security_sources` variable in the fact script template ends up being empty. That's because now the `for repo` loop operates on list of lists. You should add the lists together instead of making a set of nested lists here - `(apt__host_security_sources + apt__group_security_sources + apt__security_sources)`.